lib/config.js

```diff
@@ -48,7 +48,12 @@ config.mongodb.connectOptions = {
     journal: true
   },
   server: {
-    auto_reconnect: true
+    socketOptions: {
+      autoReconnect: true, // reconnect on connection failure
+      keepAlive: 15000, // TCP keep-alive every 15 seconds (useful for VPN'd connections)
```

Comment:
This `keepAlive` value translates to `keepAliveInitialDelay` in mongodb which ends up getting passed here to node's net socket API:

https://nodejs.org/api/net.html#net_socket_setkeepalive_enable_initialdelay

Here's an explanation of TCP keep alive:

https://stackoverflow.com/questions/1480236/does-a-tcp-socket-connection-have-a-keep-alive

In short, when using keep alive on a TCP connection, it will wait X number of seconds (not set here)  after no activity on the connection, then send an empty ACK (a "keep alive probe"). If there's no reply from the server, it will send another one after `keepAliveInitialDelay` (the value set here) milliseconds. It will keep doing this until the server responds or it runs out of probes (default on linux is 9 probes).

Anyway, I don't think this is doing what we want and we should remove it and figure out if anything else needs to be set to remedy whatever problem we're seeing.